scripts/runner_test.go

```diff
@@ -0,0 +1,41 @@
+package scripts
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLookPath(t *testing.T) {
+	tests := []struct {
+		scriptFile string
+		wantErr bool
+	}{
+		{"no-such-file", true},
+		{"fetch-uj-records.sh", false},
+		{"get-uid-map.sh", false},
+		{"splunk-to-segment.sh", false},
+		{"segment-mass-uploader.sh", false},
+	}
+	for _, tt := range tests {
+		t.Run(tt.scriptFile, func(t *testing.T) {
+			assert := assert.New(t)
+			require := require.New(t)
+			got, err := LookPath(tt.scriptFile)
+			if tt.wantErr {
+				assert.Error(err)
+				return
+			}
+			require.Nil(err, "Must be able to find %v", tt.scriptFile)
```

Comment:
I think assert/require for `Error()` might be more straightforward and also follows the same path of the case that has error.
https://pkg.go.dev/github.com/stretchr/testify@v1.8.4/assert#NoErrorf